content/swift/concepts/operators/terms/basic-operators/basic-operators.md

```diff
@@ -0,0 +1,208 @@
+---
+Title: 'Basic Operators'
+Description: 'Basic operators in Swift are commonly used to evaluate, reassign, and combine values.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Operators'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+**Basic operators** in Swift are commonly used to evaluate, reassign, and combine values.
+
+## Assignment Operator
+
+The assignment operator `=` in Swift is the same as in most other languages and serves the same purpose. It is used to initialize or reassign a variable to some value. In Swift, unlike some other high-level programming languages like C or Java, the assignment operator does not return any values.
+
+```swift
+var a = 5
+// When we reference variable 'a', it returns the value int: 5.
+let z = (a = 6)
+// Compile warning that 'b' is of type '()' "
+// Output: '()'
+```
+
+Here, the `z = (a=6)` assignment leaves `z` to equal `()` once the `a=6` is evaluated.
+
+## Arithmetic Operators
+
+Swift contains the following operators that match their similar expressions in common math:
+
+- `+` for addition
+- `-` for subtraction
+- `*` for multiplication
+- `/` for division
+- `%` for modulus or remainder division
+
+These work as you would expect for all mathematical operations, however, the `+` operator can also be used to concatenate two strings together such as `"Operators" + " in Swift" = "Operators in Swift"`.
+
+## Comparison Operators
+
+These operators are used to return `Bool` values for use in conditional statements. These control logic flow in loops or if statements for example. The basic comparison operators are below:
+
+- `==` equal to
+- `!=` not equal to
+- `>` greater than
+- `>=` greater than or equal to
+- `<` less than
+- `<=` less than or equal to
+
+The following code snippet shows a use case of comparison operators in a `while` loop that will :
+
+```swift
+var countUp = 0
+var countDown = 10
+
+while countUp < countDown {
+  countUp = countUp + 1
+  countDown = countDown - 1
+}
+```
+
+## Logical Operators
+
+Logical Operators are used to evaluate or modify the value of Boolean values and are especially useful in control flow logic. In Swift, there are three operators:
+
+- `!` Logical NOT
+- `&&` Logical AND
+- `||` Logical OR
+
+Logical NOT(`!`) operator appears directly before a Boolean value with no spaces. This operator inverts the value of the Boolean so that `true` becomes `false` and vice versa.
```

Comment:
```suggestion
Logical NOT (`!`) operator appears directly before a Boolean value with no spaces. This operator inverts the value of the Boolean so that `true` becomes `false` and vice versa.
```